src/definitions/modules/calendar.js

```diff
@@ -745,20 +745,26 @@ $.fn.calendar = function(parameters) {
           minDate: function (date) {
             date = module.helper.sanitiseDate(date);
             if (settings.maxDate !== null && settings.maxDate <= date) {
-              module.verbose('Unable to set minDate variable bigger that maxDate variable', date, settings.maxDate);
-            } else {
-              module.setting('minDate', date);
-              module.set.dataKeyValue(metadata.minDate, date);
+              if (!settings.autoAdjustDateEdges) {
+                module.error('Unable to set minDate variable bigger than maxDate variable', date, settings.maxDate);
+                return;
+              }
+              module.set.maxDate(date);
```

Comment:
My understanding of new behavior: If the new option is enabled, minDate and maxDate both use `date`.
What use cases can be considered for such behavior ?

Reply:
The same date is only set for both in case the mindate would otherwise be set after the maxdate (or the maxdate before an existing mindate) which is an invalid range. To keep the range valid, the date is set to the same date.  
This is for cases where, for example, the min/max date is set by a user interaction and has to keep a valid range. 

The only other possibility I see is to set the other date to null instead of the same date.

Better approach then?

Reply:
Ah, I see.